api/turing/service/ensembling_job_service.go

```diff
@@ -100,6 +102,14 @@ func (s *ensemblingJobService) List(options EnsemblingJobListOptions) (*Paginate
 		query = query.Where("project_id = ?", options.ProjectID)
 	}
 
+	if options.EnsemblerID != nil {
+		query = query.Where("ensembler_id = ?", options.EnsemblerID)
+	}
+
+	if options.Search != nil && len(*options.Search) > 0 {
+		query = query.Where("name like ?", fmt.Sprintf("%%%s%%", *options.Search))
```

Comment:
This probably requires indexing of the database on the `name` field.

Reply:
I prefer to get back to this when it cause us performance issues